### PR TITLE
ovirt_disk: fix vm names search and allow attaching to vm

### PR DESCRIPTION
**Pull request by mnecas**

### CODE REVIEW

**Comment by mwperina** [comment]
Please add commit message, it's hard to review when it's not clear what should be fixed

**Review by mwperina — APPROVED** [review]
+1

File: plugins/modules/ovirt_disk.py [hunk 1]
```diff
@@ -697,7 +697,7 @@ def searchable_attributes(module):
     attributes = {
         'name': module.params.get('name'),
         'Storage.name': module.params.get('storage_domain'),
-        'vm_names': module.params.get('vm_name'),
+        'vm_names': module.params.get('vm_name') if module.params.get('state') != 'attached' else None,
```
Comment (barpavel) [inline plugins/modules/ovirt_disk.py:700]: Minor: in all places "module.params['state']" syntax is used.

**Review by barpavel — APPROVED** [review]

File: plugins/modules/ovirt_disk.py [hunk 2]
```diff
@@ -765,7 +765,7 @@ def searchable_attributes(module):
...
```
Comment (barpavel) [inline plugins/modules/ovirt_disk.py:768]: Now after looking at it again, have 1 question, should this `vm_name` filtering by `state` should be be done here on when used according to the flow?
I mean this `searchable_attributes()` looks like a pretty generic method, that has a potential to be used in multiple places.

Comment (mwperina) [reply plugins/modules/ovirt_disk.py:770]: If it's worth, please do that in followup patch

**Pull request merged**